### PR TITLE
Describe SC optional gas price

### PR DESCRIPTION
**Pull request by velzevur**
Complementary RP to aeternity/aeternity#3077

### CODE REVIEW

File: node/api/channels_api_usage.md [hunk 1]
```diff
@@ -23,13 +23,19 @@ parties_.
...
 API calls.
 
-Although no off-chain transactions consume gas nor require
-fees, all on-chain transactions come with a fee. The value of the fee can be
-set by the client that initiates the action, ex. a deposit. If not provided,
-the FSM will calculate it for the client: it will multiply the minimum gas
-required for the transaction by the node's setting for `min_miner_gas_price`.
-Note that using the `min_miner_gas_price` could be too low or too high
-according to dynamically changing miner expectations for gas price.
+Although no off-chain transactions consume gas nor they require any fees, all
```
Comment (uwiger) [inline node/api/channels_api_usage.md]:
```suggestion
Although no off-chain transactions consume gas or require any fees, all
```

File: node/api/channels_api_usage.md [hunk 2]
```diff
@@ -23,13 +23,19 @@ parties_.
...
+on-chain transactions come with a fee. The value of the fee can be set by the
+client that initiates the action, ex. a deposit. The FSM could also calculate
+it for the client: it will multiply the minimum gas required for the
+transaction by the gas price. The gas price could optionally be specified by
+the client. If not - the node's setting for `min_miner_gas_price` instead.
```
Comment (uwiger) [inline node/api/channels_api_usage.md]:
```suggestion
the client. If not - the node's setting for `min_miner_gas_price` is used instead.
```

File: node/api/channels_api_usage.md [hunk 3]
```diff
@@ -23,13 +23,19 @@ parties_.
...
+Note that relying on the `min_miner_gas_price` could result in fee being
```
Comment (uwiger) [inline node/api/channels_api_usage.md]:
```suggestion
Note that relying on the `min_miner_gas_price` could result in the fee being
```

**Comment by velzevur** [comment]
Snapshot is completely missing from the `node/api/channel_ws_api.md ` but it is added in pr #448 where the gas price will be specified

**Comment by velzevur** [comment]
Rebased and added snapshot's `gas_price`

**Pull request merged**